app/views/RoomsListView/components/ServersList.tsx

```
@@ -1,27 +1,25 @@
-import { RootEnum, TServerModel } from '../../../definitions';
+import { RootEnum } from '../../../definitions';
 import I18n from '../../../i18n';
 import { TOKEN_KEY } from '../../../lib/constants/keys';
 import database from '../../../lib/database';
 import { useAppSelector } from '../../../lib/hooks/useAppSelector';
-import { removeServer } from '../../../lib/methods/logout';
 import EventEmitter from '../../../lib/methods/helpers/events';
 import { goRoom } from '../../../lib/methods/helpers/goRoom';
-import { showConfirmationAlert } from '../../../lib/methods/helpers/info';
 import { localAuthenticate } from '../../../lib/methods/helpers/localAuthentication';
 import log, { events, logEvent } from '../../../lib/methods/helpers/log';
```

ESLint

[error] 'log' is defined but never used.

(@typescript-eslint/no-unused-vars)